CONTRIBUTING.md

```diff
@@ -2,6 +2,101 @@
 
 We love improvements to our tools! EDGI has general [guidelines for contributing](https://github.com/edgi-govdata-archiving/overview/blob/master/CONTRIBUTING.md) to all of our organizational repos.
 
+
 ## Submitting Web Monitoring Issues
 
 Issues that are project-wide, or relate heavily to the interaction between different components, should be added to our [Web Monitoring issue queue](https://github.com/edgi-govdata-archiving/web-monitoring/issues). Component-specific issues should be added to their respective repository.
+
+
+## Code Style / Best Practices
+
+The following are recommended code styling and best practices for the web-monitoring-ui repository. We also have best practices for related to all the [web-monitoring] project (https://github.com/edgi-govdata-archiving/web-monitoring/blob/master/CONTRIBUTING.md) repo.
+
+
+## UI - Support
+
+### Browser Support:
+
+Last two of every major browser (Chrome, Safari, Firefox, Edge) except IE
+
+
+### ES6+ allowed everywhere:
+
+ES6 and above is allowed. A feature is allowed if it has been formally published or is a Stage 4 proposal.  Otherwise, it is not allowed.
+
+
+## UI - Code Style / Best Practices
+
+### CSS Methodologies:
+
+Recommend BEM as much as possible (as opposed to (OOCSS, SMACSS, ITCSS)
```

Comment:
Need a closing `)` at the end of this line.

Reply:
fixed!